tests/suites/test_suite_rsa.function

```diff
@@ -29,20 +32,65 @@ void rsa_invalid_param( )
                                          invalid_hash_id ),
                 MBEDTLS_ERR_RSA_INVALID_PADDING );
 
+    TEST_EQUAL( mbedtls_rsa_pkcs1_sign(&ctx, NULL,
+                                       NULL, md_alg_none,
+                                       buf_len,
+                                       NULL, buf),
+                MBEDTLS_ERR_RSA_BAD_INPUT_DATA );
+
+    TEST_EQUAL( mbedtls_rsa_pkcs1_verify(&ctx, md_alg_none,
+                                         buf_len,
+                                         NULL, buf),
+                MBEDTLS_ERR_RSA_BAD_INPUT_DATA );
+
 #if !defined(MBEDTLS_PKCS1_V15)
     TEST_EQUAL( mbedtls_rsa_set_padding( &ctx,
                                          MBEDTLS_RSA_PKCS_V15,
                                          MBEDTLS_MD_NONE ),
                 MBEDTLS_ERR_RSA_INVALID_PADDING );
 #endif
 
+#if defined( MBEDTLS_PKCS1_V15 )
```

Comment:
Minor style guide change, `defined` is exempted from the parentheses space rules:
```suggestion
#if defined(MBEDTLS_PKCS1_V15)
```

Reply:
Made the change